app.json

```diff
@@ -35,9 +35,17 @@
     "CYBERSOURCE_ACCESS_KEY": {
       "description": "CyberSource Access Key"
     },
+    "CYBERSOURCE_INQUIRY_LOG_NACL_ENCRYPTION_KEY": {
+      "description": "The public key to encrypt export results with",
+      "required": false
+    },
```

Comment:
Am I correct that this key is one that we generate, and not one that comes from Cybersource? If so, can we make that clear somehow?

Reply:
Can you add that it needs to be 32 bytes, base64 encoded?